test/test_cuda.py

```diff
@@ -2585,7 +2585,12 @@ def test_rpow(self):
         _TestTorchMixin._test_rpow(self, lambda x: x.cuda())
 
     def test_int_pow(self):
-        _TestTorchMixin._test_int_pow(self, lambda x: x.cuda())
+        _TestTorchMixin._test_integral_pow(self, lambda x: x, torch.int8, (-3, 4))
```

Comment:
`lambda x: x` supposed to be `lambda x: x.cuda()` right?

Reply:
Replaced this test with @torchtest.test_all_device_types()